### PR TITLE
Upgrading pom version from 1.0.5 to 1.0.6

### PR DESCRIPTION
**Pull request by gashutos**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [1.0.6] - 2022-08-16
+### Added
+- Basic auth support for OAuth2. We need to add Base64 encoded Basic Auth header while making OAuth2 request in OAuth2 retrieve token scheme.
+- Removed ConnectorContext from validate creds request
+- Made api version nullable
+- Removed api version from connector runtime settings and added IsSandboxAccount
```
Comment (sh9694) [inline ChangeLog.md:14]: Can we reframe this, this has been added in example

**Review by sh9694 — COMMENTED** [review]
I think there was one more change Dilip has added for retry which also we can add and push as part of this

File: ChangeLog.md (hunk 1 above)
Comment (gashutos) [reply ChangeLog.md]: I have removed sandbox account phrase, its not a feature imo

**Comment by gashutos** [comment]
> can
Added it.,

**Review by sh9694 — APPROVED** [review]

**Pull request merged**